operator/api/auto/v1alpha1/update_types.go

```diff
@@ -39,10 +39,13 @@ const (
 type UpdateSpec struct {
 	// WorkspaceName is the workspace to update.
 	WorkspaceName string `json:"workspaceName,omitempty"`
-	StackName     string `json:"stackName,omitempty"`
-
+	// Specify the Pulumi stack to select for the update.
+	StackName string `json:"stackName,omitempty"`
+	// Type of the update to perform.
 	Type UpdateType `json:"type,omitempty"`
-
+	// TTL for a completed update object.
+	// +optional
+	TtlAfterCompleted *metav1.Duration `json:"ttlAfterCompleted,omitempty"`
```

Comment:
Should we align naming to be similar to Jobs? `.ttlSecondsAfterFinished` Otherwise, we should be more specific about the accepted units of time here. Is it seconds or hours etc

Reply:
It is a metav1.Duration, not an integer.  I took a bit of time to look at newer APIs from kube SIGs, and they seem to prefer that the fields be like this, as opposed to the older style seen in Job.

I could use "finished" rather than "completed", but note that we have a `Completed` status condition.

Reply:
I agree with using `Completed`, as that's the nomenclature we're using for conditions. I'm more familiar with ttl fields using `int32` as their values, but I'm not opposed to using `time.Duration` instead.